metalsmith/utils/installguidepagifier.js

```diff
@@ -0,0 +1,63 @@
+import markdownToc from 'markdown-toc';
+import resolveI18n from './resolveI18n';
+export default pageify;
+
+/**
+ *  Pageifies a single Contentful entry
+ */
+function pageify(entryRaw, i18n){
+    let file;
+
+    let entry = resolveI18n(entryRaw);
+    let fields = entry.fields;
+    console.log('fields ', fields);
+
+    if ("programTemplate" == entry.sys.contentType.sys.id && fields.installGuide != undefined) {
+        
+        // if(!fields.globallyInstallable){
+        //     // Don't show docs for non-globally installable things
+        //     return null;
+        // }
+        
+
+        file = {
+            title: fields.name + " Setup Guide",
+            highlights: fields.summary,
+            contents: fields.installGuide,
+            markdownToc: markdownToc(fields.installGuide),
+            tags: fields.tags,
+            slug: "program/" + fields.slug + "/setup-guide",
+
+            fields: fields,
+            id: entry.sys.id,
+            sectionType: "successArticle",
```

Comment:
Should install guides be considered to part of the success center or the developer center?